### PR TITLE
Remove unused parameter in StudentProfilePage.java #6439

### LINKED ISSUE #6439: Remove unused parameter in StudentProfilePage.java
In `teammates.test.pageobjects.StudentProfilePage#editProfileThroughUi`, the first parameter `fileName` is not used and thus can be removed.

### PR DESCRIPTION
**Pull request by PratyushTalreja**
Fixes #6439

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/StudentProfilePage.java [hunk 1]
```diff
@@ -150,7 +150,7 @@ public void selectGender(String gender) {
         }
     }
 
-    public void editProfileThroughUi(String fileName, String shortName, String email, String institute,
+    public void editProfileThroughUi(String shortName, String email, String institute,
```
Comment (AnaghHegde) [inline src/test/java/teammates/test/pageobjects/StudentProfilePage.java]: When you change the source file corresponding test files also will be affected. You need to change the test file also. That's why the build is failing.

**Comment by JosephineKwa** [comment]
Hi @PratyushTalreja, this issue doesn't merely involve the removal of the unused param in the method declaration, we also have to update the other parts of the code that use this method. A straightforward way would be to right-click on the method and refactor the method signature. That way eclipse will automatically handle the updating for us, rather than having to do it manually.

File: src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java [hunk 2]
```diff
@@ -113,20 +113,20 @@ private void testActions() {
...
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "American",
-                                         "other", "this is enough!$%&*</>");
+        profilePage.editProfileThroughUi("short.name", "e@email.tmt", "inst", "American", "other",
+                                         "this is enough!$%&*</>");
         profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "American",
                                           "other", "this is enough!$%&*</>");
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "American",
-                                        "female", "this is enough!$%&*</>");
+        profilePage.editProfileThroughUi("short.name", "e@email.tmt", "inst", "American", "female",
+                                        "this is enough!$%&*</>");
```
Comment (JosephineKwa) [inline src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java]: Do align the start of this line with the start of the params in the line above, i.e.
```
profilePage.editProfileThroughUi("short.name", ...
                                 "this is ...
```

Comment (PratyushTalreja) [inline src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java]: Okay @JosephineKwa . I have made the changes to the line number 129.

**Pull request merged**